### PR TITLE
increase the size of txn to support vault

### PR DESCRIPTION
**Pull request by malizz**
### Description
increase the size of txn to support vault

### PR Checklist

* [x] updated test coverage
* [ ] external facing docs updated
* [x] not a security concern

### CODE REVIEW

**Comment by raskchanky** [comment]
Thank you for this!

**Comment by freddygv** [comment]
Thank you for addressing this @malizz ! 

Could you also add a test case with 128 `TxnOps` that doesn't yield an error? That way we be more confident that the intended limit is applying

File: .changelog/14599.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+api: Increase max operations size of a transaction for requests to /v1/txn
```
Comment (freddygv) [inline .changelog/14599.txt:2]:
```suggestion
api: Increase max number of operations inside a transaction for requests to /v1/txn (128)
```

**Review by freddygv — APPROVED** [review]
Awesome, just left one minor suggestion for the changelog entry. 

Primarily so that the changelog shows the new limit.

**Pull request merged**